src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7192,11 +7224,11 @@ namespace UnitarySystems {
                 }
                 input_specs.dehumidification_control_type = dehumidControlType;
 
-                std::string runOnSensibleLoad("Yes");
+                std::string runOnSensibleLoad("YES");
```

Comment:
Blank inputs were not getting set correctly.

Reply:
Literal string variables should be `constexpr std::string_view`, not `std::string`. Want to avoid constructing a std::string object on every call.

Reply:
Well, this is a default value and will change if the input field is present and not blank.

    std::string runOnSensibleLoad("YES");
    if (fields.find("run_on_sensible_load") != fields.end()) {
        runOnSensibleLoad = UtilityRoutines::MakeUPPERCase(AsString(fields.at("run_on_sensible_load")));
    }

Reply:
So you are using a local string whose value is either "YES" or "NO" instead of a bool? 😱  A (much) better thing to do is:
 
```
bool runOnSensibleLoad = true;
if (fields.find("run_on_sensible_load") != fields.end()) {
    runOnSensibleLoad = UtilityRoutines::SameString(fields.at("run_on_sensible_load"), "YES");
}
```

Reply:
I like that one! Done.